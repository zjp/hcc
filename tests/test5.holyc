123423452345 
123452345234523452345234523452345345
