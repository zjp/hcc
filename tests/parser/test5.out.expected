int a;
intptr b;
int c;
int fn(){
	a = 1;
	c = 9;
	b = ^c;
	b[0] = a;
}
